2021-05-27 08:32:46,082 INFO    MainThread:17895 [wandb_setup.py:_flush():69] setting env: {}
2021-05-27 08:32:46,082 INFO    MainThread:17895 [wandb_setup.py:_flush():69] setting login settings: {}
2021-05-27 08:32:46,083 INFO    MainThread:17895 [wandb_init.py:_log_setup():336] Logging user logs to /opt/ml/gitclone/p4-mod-model_diet/wandb/run-20210527_083246-1l0erg5x/logs/debug.log
2021-05-27 08:32:46,083 INFO    MainThread:17895 [wandb_init.py:_log_setup():337] Logging internal logs to /opt/ml/gitclone/p4-mod-model_diet/wandb/run-20210527_083246-1l0erg5x/logs/debug-internal.log
2021-05-27 08:32:46,083 INFO    MainThread:17895 [wandb_init.py:init():369] calling init triggers
2021-05-27 08:32:46,083 INFO    MainThread:17895 [wandb_init.py:init():374] wandb.init called with sweep_config: {}
config: {'img_size': 384, 'init_lr': 0.00800567700698709}
2021-05-27 08:32:46,083 INFO    MainThread:17895 [wandb_init.py:init():418] starting backend
2021-05-27 08:32:46,083 INFO    MainThread:17895 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-05-27 08:32:46,093 INFO    MainThread:17895 [backend.py:ensure_launched():132] starting backend process...
2021-05-27 08:32:46,099 INFO    MainThread:17895 [backend.py:ensure_launched():136] started backend process with pid: 17904
2021-05-27 08:32:46,100 INFO    MainThread:17895 [wandb_init.py:init():423] backend started and connected
2021-05-27 08:32:46,102 INFO    MainThread:17895 [wandb_init.py:init():465] updated telemetry
2021-05-27 08:32:46,102 INFO    MainThread:17895 [wandb_init.py:init():484] communicating current version
2021-05-27 08:32:46,860 INFO    MainThread:17895 [wandb_init.py:init():489] got version response 
2021-05-27 08:32:46,860 INFO    MainThread:17895 [wandb_init.py:init():497] communicating run to backend with 30 second timeout
2021-05-27 08:32:47,089 INFO    MainThread:17895 [wandb_init.py:init():522] starting run threads in backend
2021-05-27 08:32:47,130 INFO    MainThread:17895 [wandb_run.py:_console_start():1565] atexit reg
2021-05-27 08:32:47,131 INFO    MainThread:17895 [wandb_run.py:_redirect():1412] redirect: SettingsConsole.REDIRECT
2021-05-27 08:32:47,131 INFO    MainThread:17895 [wandb_run.py:_redirect():1417] Redirecting console.
2021-05-27 08:32:47,133 INFO    MainThread:17895 [wandb_run.py:_redirect():1479] Redirects installed.
2021-05-27 08:32:47,133 INFO    MainThread:17895 [wandb_init.py:init():546] run started, returning control to user process
2021-05-27 08:33:37,476 INFO    MainThread:17895 [wandb_run.py:_atexit_cleanup():1535] got exitcode: 255
2021-05-27 08:33:37,476 INFO    MainThread:17895 [wandb_run.py:_restore():1507] restore
2021-05-27 08:33:39,482 INFO    MainThread:17895 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 717
  total_bytes: 717
}

2021-05-27 08:33:39,757 INFO    MainThread:17895 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 717
  total_bytes: 717
}

2021-05-27 08:33:40,125 INFO    MainThread:17895 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 717
  total_bytes: 3774
}

2021-05-27 08:33:40,878 INFO    MainThread:17895 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3774
  total_bytes: 3774
}

2021-05-27 08:33:40,980 INFO    MainThread:17895 [wandb_run.py:_wait_for_finish():1657] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3774
  total_bytes: 3774
}

2021-05-27 08:33:41,082 INFO    MainThread:17895 [wandb_run.py:_wait_for_finish():1657] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 3774
  total_bytes: 3774
}

2021-05-27 08:33:42,176 INFO    MainThread:17895 [wandb_run.py:_show_files():1879] logging synced files
